guide/app_ssystem_ini.tex

```diff
@@ -337,7 +335,7 @@ \subsubsection{Minor Planet section}
 \subsubsection{Comet section}
 \label{sec:ssystem.ini:Comet}
 
-Comets are tiny, and their outgassing and close approaches to the
+Comets are tiny, and their outgassing and close approaches to Sun and
```

Comment:
The orbiting around the Sun does not change the orbital elements which model a perfect 2-body system of massive sun and relatively mass-less comet. It's outgassing ("jet propulsion" of some kind), radiation pressure, and gravitational interaction with the planets which cause the changes. Please revert this one.

Reply:
Indeed, it's a bit confusing ; I've rephrased it, see upcoming commit .

As a side note, isn't this paragraph part of the solution of #1299 ?

Reply:
I have reviewed it here, and the paragraph is good as it was.